src/components/Tokens/TokenDetails/MobileBalanceSummaryFooter.tsx

```diff
@@ -15,7 +15,7 @@ const Wrapper = styled.div`
   border: 1px solid ${({ theme }) => theme.backgroundOutline};
   background-color: ${({ theme }) => theme.backgroundSurface};
   border-radius: 20px 20px 0px 0px;
-  bottom: 56px;
+  bottom: 50px;
```

Comment:
I think this value should be 52px. (I'm looking at the bottom element height value.)

As a general rule we try to make sure all px values are divisible by 4.

Reply:
good catch, i think the border messed this up. just fixed it